data/pathfinder/homebrew/final_fantasy/ff_abilities_unstable.lst

```diff
@@ -75,7 +75,7 @@ Evasion				KEY:Beastmaster ~ Evasion								CATEGORY:Special Ability	TYPE:Beastm
ge if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half damage, the animal companion takes no damage if it makes a successful saving throw and half saving throw fails. ng throw fails. ng throw fails.ng throw fails. ng throw fails. ng throw fails.
```

Comment:
I think this may be a first. Someone fixing a typographical error in an extremely error laced file that is not used for anything other than to stage in-progress work. Every little bit helps I suppose.